contracts/IICO.sol

```diff
@@ -269,6 +270,7 @@ contract IICO {
     function search(uint _maxVal, uint _nextStart) constant public returns(uint nextInsert) {
         uint next = _nextStart;
         bool found;
+        if (_maxVal == 0) _maxVal = INFINITY;
```

Comment:
_maxVal=0 does not mean INFINITY, to mean infinity, you can use the fallback function.

Reply:
Or you can pass in `-2`.

Reply:
I only add this to comply whit the function description:

@param _maxVal The maximum valuation given by the contributor. Or INFINITY if no maximum valuation is given.